src/lib/ydocker/main_dialog.rb

```diff
@@ -332,10 +354,25 @@ def update_images_buttons
 
     def update_containers_buttons
       is_something_selected = !Yast::UI.QueryWidget(:containers_table, :SelectedItems).empty?
-      [:container_inject, :container_changes, :container_stop, :container_kill, :container_commit].each do |item|
+      [:container_kill, :container_commit].each do |item|
         Yast::UI.ChangeWidget(item, :Enabled, is_something_selected)
       end
+      if is_something_selected
+        status = selected_container.info['State']['Status']
+        Yast::UI.ChangeWidget(:container_start, :Enabled, status != 'running')
+        Yast::UI.ChangeWidget(:container_changes, :Enabled, status == 'running')
+        Yast::UI.ChangeWidget(:container_inject, :Enabled, status == 'running')
+        Yast::UI.ChangeWidget(:container_stop, :Enabled, status == 'running')
```

Comment:
a bit duplicite, maybe use instead `running = selected_container.info['State']['Status'] == "Running"` and then use `running` or `!running` as all of them check same status.